Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -287,8 +287,9 @@ open class _ComposerVC<ExtraData: ExtraDataTypes>: _ViewController,
         composerView.confirmButton.isEnabled = !content.isEmpty
 
         let isAttachmentButtonHidden = !content.isEmpty || channelConfig?.uploadsEnabled == false
-        let isCommandsButtonHidden = !content.isEmpty
+        let isCommandsButtonHidden = !content.isEmpty || channelConfig?.commands?.isEmpty == true
```

Comment:
This is possibly nit, but here we go :D 

This will fail if `channelConfig?.commands?` is `nil`, I just took a look and you can make `commands` property not optional since it either get parsed or is assigned with empty Array...